dice/core/symbol.py

```diff
@@ -53,7 +53,7 @@ def generate(self):
         """
         Generate a random bytes string.
         """
-        cnt = int(random.expovariate(0.1))
+        cnt = int(random.weibullvariate(65535, 1))
```

Comment:
Is there any reason why you set this specific (alpha, beta) pairs? And is there any reason to set different values for different functions?